### PR TITLE
Test comment API

### PR DESCRIPTION
**Pull request by mildas**

### CODE REVIEW

**Comment by mildas** [comment]
comment

**Comment by mildas** [comment]
comment

**Comment by mildas** [comment]
comment

**Comment by mildas** [comment]
$output

**Comment by mildas** [comment]
Findings:  Profile ospp:    Rule directory_access_var_log_audit removed from  ospp profile.    Part of rhel7 datastream.  Profile ncp:    Part of rhel7 datastream.    NCP profile extends changed OSPP profile  Profile cui:    Part of rhel7 datastream.    CUI profile extends changed OSPP profileRecommended tests to execute:  build_product rhel7  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui

**Comment by mildas** [comment]
Findings:  Profile ospp:    Rule directory_access_var_log_audit removed from  ospp profile.    Part of rhel7 datastream.  Profile ncp:    NCP profile extends changed OSPP profile    Part of rhel7 datastream.  Profile cui:    CUI profile extends changed OSPP profile    Part of rhel7 datastream.Recommended tests to execute:  build_product rhel7  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp

**Comment by mildas** [comment]
Findings:<br />  Profile ospp:<br />    Part of rhel7 datastream.<br />    Rule directory_access_var_log_audit removed from  ospp profile.<br />  Profile ncp:<br />    NCP profile extends changed OSPP profile<br />    Part of rhel7 datastream.<br />  Profile cui:<br />    CUI profile extends changed OSPP profile<br />    Part of rhel7 datastream.<br />Recommended tests to execute:<br />  build_product rhel7<br />  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp<br />  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui<br />  test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp<br />

**Comment by mildas** [comment]
Findings:<br nbsp;/> nbsp; nbsp;Profile nbsp;ospp:<br nbsp;/> nbsp; nbsp; nbsp; nbsp;Part nbsp;of nbsp;rhel7 nbsp;datastream.<br nbsp;/> nbsp; nbsp; nbsp; nbsp;Rule nbsp;directory_access_var_log_audit nbsp;removed nbsp;from nbsp; nbsp;ospp nbsp;profile.<br nbsp;/> nbsp; nbsp;Profile nbsp;ncp:<br nbsp;/> nbsp; nbsp; nbsp; nbsp;NCP nbsp;profile nbsp;extends nbsp;changed nbsp;OSPP nbsp;profile<br nbsp;/> nbsp; nbsp; nbsp; nbsp;Part nbsp;of nbsp;rhel7 nbsp;datastream.<br nbsp;/> nbsp; nbsp;Profile nbsp;cui:<br nbsp;/> nbsp; nbsp; nbsp; nbsp;CUI nbsp;profile nbsp;extends nbsp;changed nbsp;OSPP nbsp;profile<br nbsp;/> nbsp; nbsp; nbsp; nbsp;Part nbsp;of nbsp;rhel7 nbsp;datastream.<br nbsp;/>Recommended nbsp;tests nbsp;to nbsp;execute:<br nbsp;/> nbsp; nbsp;build_product nbsp;rhel7<br nbsp;/> nbsp; nbsp;test_suite.py nbsp;profile nbsp;--libvirt nbsp;qemu:///system nbsp;test-suite-vm nbsp;--datastream nbsp;build/ssg-rhel7-ds.xml nbsp;ncp<br nbsp;/> nbsp; nbsp;test_suite.py nbsp;profile nbsp;--libvirt nbsp;qemu:///system nbsp;test-suite-vm nbsp;--datastream nbsp;build/ssg-rhel7-ds.xml nbsp;cui<br nbsp;/> nbsp; nbsp;test_suite.py nbsp;profile nbsp;--libvirt nbsp;qemu:///system nbsp;test-suite-vm nbsp;--datastream nbsp;build/ssg-rhel7-ds.xml nbsp;ospp<br nbsp;/>

**Comment by mildas** [comment]
**Findings:**<br />  Profile ospp:<br />- Rule directory_access_var_log_audit removed from ospp profile.<br />- Part of rhel7 datastream.<br />  Profile ncp:<br />- NCP profile extends changed OSPP profile.<br />- Part of rhel7 datastream.<br />  Profile cui:<br />- CUI profile extends changed OSPP profile.<br />- Part of rhel7 datastream.<br />**Recommended tests to execute:**<br />- build_product rhel7<br />- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp<br />- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp<br />- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui<br />

**Comment by mildas** [comment]
**Findings:**      Profile ospp:- Rule directory_access_var_log_audit removed from ospp profile.- Part of rhel7 datastream.  Profile ncp:- NCP profile extends changed OSPP profile.- Part of rhel7 datastream.  Profile cui:- CUI profile extends changed OSPP profile.- Part of rhel7 datastream.**Recommended tests to execute:**- build_product rhel7- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp

**Comment by mildas** [comment]
**Findings:**  
    Profile ospp:- Rule directory_access_var_log_audit removed from ospp profile.- Part of rhel7 datastream.  Profile ncp:- NCP profile extends changed OSPP profile.- Part of rhel7 datastream.  Profile cui:- CUI profile extends changed OSPP profile.- Part of rhel7 datastream.**Recommended tests to execute:**- build_product rhel7- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp

**Comment by mildas** [comment]
**Findings:**
  Profile ospp:- Rule directory_access_var_log_audit removed from ospp profile.- Part of rhel7 datastream.  Profile ncp:- Part of rhel7 datastream.- NCP profile extends changed OSPP profile.  Profile cui:- Part of rhel7 datastream.- CUI profile extends changed OSPP profile.**Recommended tests to execute:**- build_product rhel7- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui

**Comment by mildas** [comment]
**Findings:**
  Profile ospp:
- Rule directory_access_var_log_audit removed from ospp profile.
- Part of rhel7 datastream.
  Profile ncp:
- NCP profile extends changed OSPP profile.
- Part of rhel7 datastream.
  Profile cui:
- CUI profile extends changed OSPP profile.
- Part of rhel7 datastream.
**Recommended tests to execute:**
- build_product rhel7
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp

**Comment by mildas** [comment]
**Findings:**  
  Profile ospp:  
- Part of rhel7 datastream.  
- Rule directory_access_var_log_audit removed from ospp profile.  
  Profile ncp:  
- NCP profile extends changed OSPP profile.  
- Part of rhel7 datastream.  
  Profile cui:  
- CUI profile extends changed OSPP profile.  
- Part of rhel7 datastream.  
**Recommended tests to execute:**  
- build_product rhel7  
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui  
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp  
- test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp

**Comment by mildas** [comment]
**Findings:**
Profile ospp:
 Rule directory_access_var_log_audit removed from ospp profile.
 Part of rhel7 datastream.
Profile ncp:
 Part of rhel7 datastream.
 NCP profile extends changed OSPP profile.
Profile cui:
 Part of rhel7 datastream.
 CUI profile extends changed OSPP profile.
**Recommended tests to execute:**
 build_product rhel7
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp

**Comment by mildas** [comment]
**Findings:**
Profile ospp:
 Rule directory_access_var_log_audit removed from ospp profile.
 Part of rhel7 datastream.
Profile ncp:
 NCP profile extends changed OSPP profile.
 Part of rhel7 datastream.
Profile cui:
 CUI profile extends changed OSPP profile.
 Part of rhel7 datastream.

**Recommended tests to execute:**
 build_product rhel7
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp

**Comment by mildas** [comment]
**Findings:**
Profile PROFILE_1:
 Rule x removed from PROFILE_1 profile.
 Part of PRODUCT_1 datastream.
Profile PROFILE_2:
 PROFILE_2 profile extends changed PROFILE_1 profile.
 Part of PRODUCT_1 datastream.
Rule RULE_1:
 Part of PRODUCT_1 datastream.
 Found change in bash remediation.
Rule RULE_2:
 Part of PRODUCT_2 datastream.
 Found change in bash remediation.
Macro MACRO_1:
 MACRO_1 macro is used in these files: /linux_os/guide/services/usbguard/configure_usbguard_auditbackend/bash/shared.sh, /linux_os/guide/services/ssh/ssh_server/sshd_use_strong_macs/bash/shared.sh, /linux_os/guide/services/ss
h/ssh_server/sshd_use_priv_separation/bash/shared.sh


**Recommended tests to execute:**
 build_product rhel7
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp

**Comment by mildas** [comment]
build_product rhel8
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-rhel8-ds.xml accounts_passwords_pam_faillock_interval
build_product ol7
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol7-ds.xml accounts_passwords_pam_faillock_deny
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol7-ds.xml accounts_passwords_pam_faillock_unlock_time

**Comment by mildas** [comment]
build_product rhel8
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-rhel8-ds.xml accounts_passwords_pam_faillock_interval
build_product ol7
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol7-ds.xml accounts_passwords_pam_faillock_deny
test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate bash --datastream build/ssg-ol7-ds.xml accounts_passwords_pam_faillock_unlock_time

**Comment by mildas** [comment]
**Findings:**\nProfile ospp on rhel7:\n Rule directory_access_var_log_audit removed from ospp profile.\nProfile ncp on rhel7:\n NCP profile extends changed OSPP profile.\nProfile cui on rhel7:\n CUI profile extends changed OSPP profile.\n\n**Recommended tests to execute:**\n build_product rhel7\n test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui\n test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp\n test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp\n

**Comment by mildas** [comment]
**Findings:**
Profile ospp on rhel7:
 Rule directory_access_var_log_audit removed from ospp profile.
Profile ncp on rhel7:
 NCP profile extends changed OSPP profile.
Profile cui on rhel7:
 CUI profile extends changed OSPP profile.

**Recommended tests to execute:**
 build_product rhel7
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ospp
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml cui

**Comment by mildas** [comment]
**Findings:**
Rule audit_rules_time_stime:
 Ansible remediation changed.
Rule audit_rules_kernel_module_loading:
 Ansible remediation changed.
Rule audit_rules_time_clock_settime:
 Ansible remediation changed.
Rule audit_rules_time_adjtimex:
 Ansible remediation changed.
Rule audit_rules_time_settimeofday:
 Ansible remediation changed.
Macro ansible_audit_augenrules_add_syscall_rule:
 In Ansible remediation for audit_rules_kernel_module_loading.
 In Ansible remediation for audit_rules_time_stime.
 In Ansible remediation for audit_rules_time_clock_settime.
 In Ansible remediation for audit_rules_time_adjtimex.
 In Ansible remediation for audit_rules_time_settimeofday.
Macro ansible_audit_auditctl_add_syscall_rule:
 In Ansible remediation for audit_rules_kernel_module_loading.
 In Ansible remediation for audit_rules_time_stime.
 In Ansible remediation for audit_rules_time_clock_settime.
 In Ansible remediation for audit_rules_time_adjtimex.
 In Ansible remediation for audit_rules_time_settimeofday.

**Recommended tests to execute:**
 build_product ol7
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_stime
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_kernel_module_loading
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_clock_settime
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_adjtimex
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_settimeofday

**Comment by mildas** [comment]
**Findings:**
Rule audit_rules_time_adjtimex:
 Ansible remediation newly added.
Rule audit_rules_time_settimeofday:
 Ansible remediation newly added.
Rule audit_rules_time_stime:
 Ansible remediation newly added.
Rule audit_rules_time_watch_localtime:
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product ol7
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_adjtimex
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_settimeofday
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_stime
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml audit_rules_time_watch_localtime

**Comment by mildas** [comment]
**Changes identified:**
Rule kernel_module_ipv6_option_disabled:
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product rhel8
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-rhel8-ds.xml kernel_module_ipv6_option_disabled

**Comment by mildas** [comment]
**Changes identified:**
Rule kernel_module_ipv6_option_disabled:
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product rhel8
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-rhel8-ds.xml kernel_module_ipv6_option_disabled

**Comment by mildas** [comment]
**Changes identified:**
Rule kernel_module_ipv6_option_disabled:
 Ansible remediation newly added.

**Recommended tests to execute:**
 build_product rhel8
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-rhel8-ds.xml kernel_module_ipv6_option_disabled

File: content_test_filtering.py [hunk 1]
```diff
@@ -51,7 +54,8 @@
     list_of_tests = connect_to_labels.get_labels(tests)
 
     if list_of_tests:
-        logger.info("List of tests to run:\n%s", "\n".join(list_of_tests))
+        logger.info("List of tests to run:")
+        print("\n".join(list_of_tests))
```
Comment (ggbecker) [inline content_test_filtering.py:58]: you can probably remove this print

Comment (ggbecker) [reply content_test_filtering.py:58]: or revert the change

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # content-test-filtering
 Purpose of this project is automatic tests filtering for project [ComplianceAsCode](https://github.com/ComplianceAsCode/content/).
+test
```
Comment (ggbecker) [inline README.md:3]: remove line

**Comment by ggbecker** [comment]
I thought this was the pull request that introduced the feature to edit instead of adding new comments to PR  :)

**Comment by mildas** [comment]
@ggbecker No, sorry for the confusion :-D This PR was created just to test if sending comments works as expected.

**Pull request closed without merging**